src/Symfony/Component/Validator/Constraints/ChoiceValidator.php

```diff
@@ -55,6 +59,13 @@ public function validate($value, Constraint $constraint)
                 throw new ConstraintDefinitionException('The Choice constraint expects a valid callback.');
             }
             $choices = $choices();
+        } elseif (is_a($constraint->choices, \BackedEnum::class, true)) {
+            $choices = array_map(
+                function (\BackedEnum $enum) {
+                    return $enum->value;
```

Comment:
you might be able to use array_column instead

Reply:
TIL we can use `array_column` with an array of objects! 🤯😍